src/components/PrettyJson/index.tsx

```diff
@@ -44,6 +60,26 @@ export const PrettyJson = ({ json }: { json: AnyObject }) => {
   );
 
   const render = (item: any, parentKey?: string): React.ReactElement => {
+    const renderValue = (item: any, key: string) => {
+      const custom = customKeyValueLinkMap?.[key];
+
+      if (custom) {
+        if (custom.condition && !custom.condition(item)) {
+          return render(item, key);
+        }
+
+        const href = custom.getHref(item, key);
+
+        return (
+          <SdsLink href={href || item} {...(href ? { target: "_blank" } : {})}>
```

Comment:
thoughts on adding an underline for links to make it clearer that it's linkable?

Reply:
@charles-sdf what do you think?

Reply:
@jeesunikim @quietbits yep im good with that! let's add em